docs/source/containerization/kubernetes.mdx

```diff
@@ -231,6 +225,16 @@ spec:
   restartPolicy: Never
 ```
 
-## Health checks
+## The health endpoint
+
+The router supports a health endpoint. You can see from the examples above how it can be used in a kubernetes deployment.
+
+If you had a router running on port 9090 on your localhost, you could exercise the health endpoint as follows:
+
+```bash
+curl "http://localhost:9090/health"
+{"status":"pass"}
+```
+
+This health endpoint can be used to check the health of the router. It will always return a status of pass if all is well.
```

Comment:
Isn't the health-check on port 8088 by default? The health-check has its own address, so we we should make that clear in these docs. Also, the content is wrong. It returns:
```
{"status":"UP"}
```

Something like:
```
If you had a router running on your localhost, with default health-check configuration, you could exercise the health endpoint as follows:

curl "http://localhost:8088/health"
```

Reply:
[998b099](https://github.com/apollographql/router/pull/1859/commits/998b09986844c5c11878d8c7344faecab382105f)